babyagi.py

```diff
@@ -18,6 +20,36 @@
 print("\033[96m\033[1m"+"\n*****OBJECTIVE*****\n"+"\033[0m\033[0m")
 print(OBJECTIVE)
 
+# Initialize SQLite database
+db_conn = sqlite3.connect("miniagi.db")
+db_cursor = db_conn.cursor()
+
+db_cursor.execute("""
+CREATE TABLE IF NOT EXISTS tasks (
```

Comment:
problem with this approach is that there's only one task list with no way to reset, right?

should probably have a namespace/identifier that is given (via args or env or whatever) such that one can build a new task list from scratch or continue from a previous one without having to manually delete their DB

Reply:
i guess the best namespace to use would be the pinecone table name